src/partials/head-scripts.hbs

```diff
@@ -1,35 +1,52 @@
-  analytics.load('{{this}}', { obfuscate: true });
-  analytics.page();
-  }}();
-</script>
+  <script>
+    !function(){var
+    analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment
+    snippet included
+    twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on","addSourceMiddleware","addIntegrationMiddleware","setAnonymousId","addDestinationMiddleware"];analytics.factory=function(e){return
+    function(){var t=Array.prototype.slice.call(arguments);t.unshift(e);analytics.push(t);return analytics}};for(var
+    e=0;e<analytics.methods.length;e++){var
+    key=analytics.methods[e];analytics[key]=analytics.factory(key)}analytics.load=function(key,e){var
+    t=document.createElement("script");t.type="text/javascript";t.async=!0;t.src="https://cdn.segment.com/analytics.js/v1/"
+    + key + "/analytics.min.js";var
+    n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(t,n);analytics._loadOptions=e};analytics._writeKey="{{this}}";analytics.SNIPPET_VERSION="4.15.2";
+    analytics.load('{{this}}', { obfuscate: true }); analytics.page(); }}();
+  </script>
 {{/with}}
 
 <script>var uiRootPath = '{{{uiRootPath}}}'</script>
 
-<link rel="preload" href="{{uiRootPath}}/font/material-icons-outlined-latin-400-normal.woff2" as="font"
-    type="font/woff2" crossorigin>
-<link rel="preload" href="{{uiRootPath}}/font/roboto-flex-latin-400-normal.woff2" as="font" type="font/woff2"
-    crossorigin>
-<link rel="preload" href="{{uiRootPath}}/font/roboto-mono-latin-500-normal.woff2" as="font" type="font/woff2"
-    crossorigin>
+<link
+  rel="preload"
+  href="{{uiRootPath}}/font/material-icons-outlined-latin-400-normal.woff2"
+  as="font"
+  type="font/woff2"
+  crossorigin
+/>
+<link
+  rel="preload"
+  href="{{uiRootPath}}/font/roboto-flex-latin-wght-normal.woff2"
```

Comment:
This change seems to have fixed the issue where some people were seeing the icon font names appear as text initially instead of the icon